frontend/src/queries/nodes/InsightViz/PropertyGroupFilters/AndOrFilterSelect.tsx

```diff
@@ -16,44 +18,50 @@ export function AndOrFilterSelect({
     topLevelFilter,
     prefix = 'Match',
     suffix = ['filter in this group', 'filters in this group'],
+    readOnly = false,
 }: AndOrFilterSelectProps): JSX.Element {
     return (
         <div className="flex items-center font-medium">
             <span className="ml-2">{prefix}</span>
-            <LemonSelect
-                className="mx-2"
-                size="small"
-                value={value}
-                onChange={(type) => onChange(type as FilterLogicalOperator)}
-                options={[
-                    {
-                        label: 'all',
-                        value: FilterLogicalOperator.And,
-                        labelInMenu: (
-                            <SelectOption
-                                title="All"
-                                description="Every single filter must match"
-                                value={FilterLogicalOperator.And}
-                                selectedValue={value}
-                            />
-                        ),
-                    },
-                    {
-                        label: 'any',
-                        value: FilterLogicalOperator.Or,
-                        labelInMenu: (
-                            <SelectOption
-                                title="Any"
-                                description="One or more filters must match"
-                                value={FilterLogicalOperator.Or}
-                                selectedValue={value}
-                            />
-                        ),
-                    },
-                ]}
-                optionTooltipPlacement={topLevelFilter ? 'bottomRight' : 'bottomLeft'}
-                dropdownMatchSelectWidth={false}
-            />
+            {readOnly && (
+                <ReadOnlyCohortField>{value === FilterLogicalOperator.Or ? 'any' : 'all'}</ReadOnlyCohortField>
```

Comment:
using the `ReadOnlyCohortField` isn't ideal here, since this is a more generic component. Instead, this should have a generic readOnly state.

@thmsobrmlr , @Twixes , what makes most sense here for insights/3000?

Reply:
@neilkakkar Can you give more context about issue here?